### DISCUSSION TITLE
QFIELDCLOUD service is down?

### DISCUSSION THREAD

**Discussion by Nioigte** [opengisch/QField#3839] [Q&A]
Hey github and qfieldcloud users,

Some hours ago, a colleague told me he wasn't able to enter into his qfieldcloud account, not knowing which was the problem (he's a starter on this software).

After hearing him, I went to check if everything was right on the QFIELDCLOUD web, and it always returned me a "CONNECTION_REFUSED" error, both in the QField app and web. Furthermore, in one specific moment, it showed me a page that said something like 'Qfieldcloud is temporary unreachable. Please try again later.'

It's just me, or the service has gone down again, like last time on mid-november? (don't remember exactly the date).

Thank you all in advance!

**Comment by suricactus** [comment]
Hi, @Nioigte . We are currently investigating very serious issue and to prevent data corruption we turned the service down. Find updates on https://status.qfield.org/incidents/60bd1643-7ade-4572-8e54-d0f7a4625cc8
